src/IntercomCompanies.php

```diff
@@ -77,6 +77,39 @@ public function getCompany($id, $options = [])
         return $this->client->get($path, $options);
     }
 
+
+    /**
+     * Returns a list of Users belonging to a single Company based on the Intercom ID.
+     *
+     * @see    https://developers.intercom.com/reference#list-company-users
+     * @param  string $id
+     * @param  array  $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getCompanyUsers($id, $options = [])
```

Comment:
It would be great if we can add this method to the documentation in the README file 😃